oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/authentication/ReactiveJwtUserDetailsCheckingAuthenticationConverterTests.java

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
Please remove this from the PR as it is something that should be considered separately.